superset/db_engine_specs/redshift.py

```diff
@@ -147,6 +151,19 @@ def _mutate_label(label: str) -> str:
         """
         return label.lower()
 
+    @classmethod
```

Comment:
This seems to be identical with the [`PostgresEngineSpec.convert_dttm()`](https://github.com/apache/superset/blob/38f5e4d490b1c3bc315d3695972541cec033bd87/superset/db_engine_specs/postgres.py#L360-L371) method which it inherits and thus this isn't needed.

Keeping the unit tests makes sense.

Reply:
but same postgres method is not getting called, when i use it?

Reply:
that's why if i remove method, test is getting failed as well

Reply:
i understand it inherits postgres, but somehow mixin is not working @john-bodley

Reply:
@gaurav7261 you're correct - @john-bodley my bad, `convert_dttm` is actually defined in `PostgresEngineSpec`, not `PostgresBaseEngineSpec` like I said a moment ago. So this change LGTM.

Reply:
@john-bodley @villebro can you help why precommit failed,what i need to do

Reply:
Did you check the details of the CI workflow? https://github.com/apache/superset/actions/runs/7226045710/job/19691266350?pr=26283

It specifies which changes are needed. Tip: you can configure `pre-commit` to do this automatically for you. Check here for details: https://github.com/apache/superset/blob/master/CONTRIBUTING.md#git-hooks

Reply:
thanks, got it, actually i was confused when it highlight the line on another function `get_cancel_query_id`

Reply:
@john-bodley please approve the run